MapboxDirections/MBRouteOptions.swift

```diff
@@ -260,22 +260,21 @@ open class RouteOptions: NSObject {
             params.append(URLQueryItem(name: "radiuses", value: accuracies))
         }
         
-        
-        if !segmentAttributes.isEmpty {
-            let segmentAttributesStrings = segmentAttributes.map {
-                String(describing: $0)
-            }.joined(separator: ",")
-            
-            
-            params.append(URLQueryItem(name: "annotations", value: segmentAttributesStrings))
+        var attributeOptions:[Attribute] = []
+        if !segmentAttributes.isEmpty && segmentAttributes.contains(where: [.speed, .expectedTravelTime, .distance].contains) {
+            attributeOptions = attributeOptions + segmentAttributes
         }
         
         if !nodeAttributes.isEmpty && nodeAttributes.contains(.openStreetMapNodeIdentifier) {
-            let segmentAttributesStrings = segmentAttributes.map {
+            attributeOptions = attributeOptions + nodeAttributes
+        }
+        
+        if !attributeOptions.isEmpty {
+            let attributesStrings = attributeOptions.map {
                 String(describing: $0)
-            }.joined(separator: ",")
+                }.joined(separator: ",")
```

Comment:
`Attribute.description` is [already a comma-separated list](https://github.com/mapbox/MapboxDirections.swift/blob/8fb56c4a5b09021e229e870b58726bd62c963198/MapboxDirections/MBAttribute.swift#L38), so you can set `attributesStrings` to `String(describing: segmentAttributes + .openStreetMapNodeIdentifier)` to get the same effect.